.gitignore

```diff
@@ -15,10 +15,11 @@
 vendor/
 
 #owncast specific
-config/config.yaml
-config/stats.json
```

Comment:
I think that would be a good route to go. Another thought too is that instead of using json files, could switch to an embedded database such as [boltdb](https://github.com/etcd-io/bbolt). Then expose the data via endpoints. I've used boltdb in several projects and have one that is in production in several business on site (actually using raspberry pi's). If you're interested in going that route, we could explore our options for an embedded database instead of flatfiles.

Reply:
Let me know your thoughts on this as doing more persistent storage opens a wide range of options. The primary one that comes to mind is sessions. A session would represent one streaming time frame and could have the ability to see stats, messages, and other details about it.

Reply:
Bolt is nice and straightforward, I've used it once previously as well.  The reason I wanted to use parsable flat files was simply to allow flexibility for people who might want to use that data for something and not need a dependency to read it.  Like stats, for example, I don't know what somebody would want to do with it, but I figured I'd make it easy to have those values accessible instead of requiring an API  to consume it.  But maybe that's solving for a problem that doesn't exist :)  It's something to think about, though, depending on how much stuff needs to be persisted going forward.

Reply:
From that perspective, it makes sense...but at the same time, if I'm hosting this I think it would be easier to consume an api than to read the json files. My thought is that most people will host this on another server aside from their other ones since it would require more processing power. Which in that instance, it would be easier to consume an api than to read the files themselves. 🤔 Like you said, though, it is something to think about and evaluate in the future.